config/config.example.toml

```diff
@@ -181,6 +181,7 @@ hash_key = "0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef"
 aci.base_url = "https://eu-test.oppwa.com/"
 adyen.base_url = "https://checkout-test.adyen.com/"
 adyen.secondary_base_url = "https://pal-test.adyen.com/"
+adyen.third_base_url = "https://ca-test.adyen.com/ca/services/DisputeService/v30/"
```

Comment:
> Can we proper naming convention stating the usecase of the newly added url?

It can be used by other connectors for other use cases in future so it is better to be like that